examples/svm/plot_custom_kernel.py

```diff
@@ -46,7 +46,7 @@ def my_kernel(X, Y):
 
 # Put the result into a color plot
 Z = Z.reshape(xx.shape)
-plt.pcolormesh(xx, yy, Z, cmap=plt.cm.Paired)
+plt.pcolormesh(xx, yy, Z, cmap=plt.cm.Paired, shading="auto")
```

Comment:
Since the shape of Z is always guaranteed to be the same as `xx` and `yy` we can be even more explicit according to https://matplotlib.org/stable/gallery/images_contours_and_fields/pcolormesh_grids.html:

```suggestion
plt.pcolormesh(xx, yy, Z, cmap=plt.cm.Paired, shading="nearest")
```

Reply:
I will work on the points given by you next time.